components/dashboard/src/settings/Preferences.tsx

```diff
@@ -118,9 +118,6 @@ export default function Preferences() {
 
                 <h3 className="mt-12">
                     Dotfiles{" "}
-                    <PillLabel type="warn" className="font-semibold mt-2 ml-2 py-0.5 px-2 self-center">
-                        Beta
-                    </PillLabel>
                 </h3>
                 <p className="text-base text-gray-500 dark:text-gray-400">Customize workspaces using dotfiles.</p>
```

Comment:
**thought**: Maybe we could also link back to the [dotfiles documentation](https://www.gitpod.io/docs/config-dotfiles) since this is also a complex topic on how it works, etc.

<img width="406" alt="Screenshot 2022-06-01 at 11 10 18 AM" src="https://user-images.githubusercontent.com/120486/171358641-dde96e36-82e3-482e-aad1-b55ab27fe468.png">